lib/cache_helper.rb

```diff
@@ -0,0 +1,12 @@
+# note, if you modify this file you have to restart the server or console
+module CacheHelper
+  
+  def skin_cache_key(skin)
+   'skins_generation/'+(skin.type.nil? ? ("site_skin") : skin.id.to_s)
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator `+`.